models/Delivery.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+const CurrencySet = require('./CurrencySet').schema;
+
+const deliverySchema = new mongoose.Schema({
+  sentOn: Date,
```

Comment:
You don't need to cast _id for this schema, so add `_id: false`

Reply:
fixed